<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="AB-LPT-HYD-108" failures="1" tests="1" name="Test(failed)(failed)" time="12.124" errors="0" timestamp="2025-01-10T18:45:56 IST">
  <testcase classname="testCases.AddEmployeeTest" name="addEmployee" time="6.409">
    <failure type="java.lang.AssertionError" message="expected [Successfully Saved] but found []">
      <![CDATA[java.lang.AssertionError: expected [Successfully Saved] but found []
at testCases.AddEmployeeTest.addEmployee(AddEmployeeTest.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- addEmployee -->
</testsuite> <!-- Test(failed)(failed) -->
